usagov/vote-gov-drupal
# Bug/vote 1411/migrate missing string translations

## Jira ticket (required)
[Insert Jira ticket link
](https://bixal-projects.atlassian.net/browse/VOTE-1411)

## Description (optional)
Added missing string translations to each languages .po file.

## Deployment and testing (required, if applicable)
### Pre-deploy steps
1. lando retune

### Testing steps
1. In the Administration panel, navigate to Configuration > Regional and Language > User Interface Translation
2. Verify the required string translations are present by filtering the string value for each language.
